app/src/main/java/org/mozilla/fenix/components/toolbar/BrowserToolbarView.kt

```diff
@@ -39,6 +51,38 @@ class BrowserToolbarView(
     val toolbarIntegration: ToolbarIntegration
 
     init {
+        view.setOnUrlLongClickListener {
+            val clipboard = view.context.getSystemService(CLIPBOARD_SERVICE) as ClipboardManager
```

Comment:
Make sure we're not showing paste in custom tabs and please make sure Copy and Paste+Go work in custom tabs before landing this :)

Reply:
Decided to move forward with just `copy` as AC brought up issues `paste & go` may have with PWAs.